server/packaging/server/server.dockerfile

```diff
@@ -36,12 +36,21 @@ LABEL org.opencontainers.image.description="Run the Parsec backend server."
 
 RUN groupadd --gid=1234 parsec && useradd --home-dir=/home/parsec --create-home --uid=1234 --gid=1234 parsec
 
+# Create parsec user and group
```

Comment:
We created the group & the user at the line 37, here we change the default user with `USER`